packages/wouter/test/use-location.test.tsx

```diff
@@ -31,16 +32,17 @@ describe("`value` first argument", () => {
     unmount();
   });
 
-  it("supports history.back() navigation", () => {
-    jest.useFakeTimers();
+  // Vitest doesn't support Jest's legacy timers.
+  it.skip("supports history.back() navigation", () => {
```

Comment:
I don’t even remember why I had to use fake timers here. Does the test pass when you remove these calls?

Reply:
I replaced `FakeTimers` with `waitFor` util from `testing-library`